src/main/groovy/io/openliberty/tools/gradle/tasks/DevTask.groovy

```diff
@@ -924,11 +943,16 @@ class DevTask extends AbstractServerTask {
                         + " dockerfile should be a valid Dockerfile");
                 }
             }
-    
+
             if (dockerRunOpts != null) {
                 setContainer(true);
                 return;
             }
+
+            if (dockerTimeout != null) {
+                setContainer(true);
+                return;
+            }
```

Comment:
If https://github.com/OpenLiberty/ci.maven/pull/907#discussion_r472243409 will be changed to not set the container option, then remove this part

Reply:
Agreed